apps/api/src/app/controllers/response-mapping/query-response-transform.interceptor.ts

```
@@ -18,6 +18,10 @@ export class QueryResponseTransformInterceptor<T> implements NestInterceptor<T,
                     throw new CoscradNotFoundException();
                 }
 
+                if (result instanceof CoscradInvalidUserInputException) {
```

At some point, we need to do some improvement on our error \ exception hierarchy. What we presently call `InternalError` is serving 2 purposes:
1. an actual internal (system) error that  should be thrown instead of returned, i.e. honest exceptions
2. A top-level wrapper for all errors created by the COSCRAD back-end (potentially anywhere in COSCRAD if error handling becomes a lib), including recursive "inner errors"